### PR TITLE
add preset for ner

### PR DESCRIPTION
**Pull request by cheungdaven**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by zhiqiangdon — APPROVED** [review]
LGTM

File: multimodal/src/autogluon/multimodal/presets.py [hunk 1]
```diff
@@ -386,6 +386,31 @@ def medium_quality_faster_inference_image_text_similarity():
 
 
 @automm_presets.register()
+def best_quality_ner():
+    return {
+        "model.names": ["ner"],
+        "model.ner.checkpoint_name": "google/flan-t5-base",
+        "env.precision": 32,
```
Comment (sxjscience) [inline multimodal/src/autogluon/multimodal/presets.py:393]: How about use `"bf16"`?

File: multimodal/src/autogluon/multimodal/presets.py [hunk 2]
```diff
@@ -386,6 +386,31 @@ def medium_quality_faster_inference_image_text_similarity():
...
+        "model.ner.checkpoint_name": "google/flan-t5-small",
+        "env.precision": 32,
+    }
+
+
+@automm_presets.register()
+def high_quality_fast_inference_ner():
+    return {
+        "model.names": ["ner"],
+        "model.ner.checkpoint_name": "bert-base-cased",
```
Comment (sxjscience) [inline multimodal/src/autogluon/multimodal/presets.py:410]: What's the performance of architectures like `deberta-v3-small`?

Comment (cheungdaven) [reply multimodal/src/autogluon/multimodal/presets.py:410]: In general, deberta-v3 works better than bert-base, but
using ddp_spawn + multiple gpus will cause an error.
using ddp + multiple gpus does not lead to good performance.

File: multimodal/src/autogluon/multimodal/presets.py (hunk 1 above)
Comment (cheungdaven) [reply multimodal/src/autogluon/multimodal/presets.py]: changed to "bf16"

File: multimodal/src/autogluon/multimodal/presets.py (hunk 2 above)
Comment (sxjscience) [reply multimodal/src/autogluon/multimodal/presets.py:416]: What type of error? Shall we fix it?

Comment (cheungdaven) [reply multimodal/src/autogluon/multimodal/presets.py:416]: cuda error, cannot find some functions from cuda.   It may be an error from pytorch lightning.

**Review by bryanyzhu — APPROVED** [review]
LGTM

**Review by FANGAreNotGnu — APPROVED** [review]
LGTM！

**Pull request merged**